src/functions-and-arrays.js

```diff
@@ -26,7 +72,16 @@ function sum() {}
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(numbersArray) {
+  if (numbersArray.length === 0){
+    return null;
+  }
+  let sum = 0;
```

Comment:
Since you previously created a function to sum numbers, you should reuse it here instead of repeating the whole logic to sum the numbers.
```
let sum = sumNumbers(numbersArray);
return sum / numbersArray.length
```